KMM_Showcase/src/main/res/layout/fragment_job_details.xml

```diff
@@ -4,6 +4,12 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <TextView
+        android:id="@+id/titleTv"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"/>
```

Comment:
@hoangnguyen92dn could you pls try to display a Toast instead in Fragment? This could lead to a conflict.

Reply:
@hoangnguyen92dn So do we still need this view in layout?